commands/peers/commands.go

```diff
@@ -0,0 +1,40 @@
+// Package peercommands implements the peer management commands
+package peercommands
+
+import (
+	"github.com/gluster/glusterd2/rest"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() rest.Routes {
+	return rest.Routes{
+		rest.Route{
+			Name:        "GetPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}",
```

Comment:
I'd like it to allow both name and uuid here. So don't think that 'peerid' just means the uuid.